release_notes_generator/action_inputs.py

```diff
@@ -38,49 +42,114 @@
 
 
 class ActionInputs:
+    """
+    A class representing the inputs provided to the GH action.
+    The class is responsible for handling the inputs provided to the GH action.
```

Comment:
You are often duplicating the information from module doc-string and class doc-string. `the inputs provided to the GH action.` is written in this class three times.

Reply:
I have removed 1 of them. The rest I kept as module and class info.